src/commands/targeting/__snapshots__/update.test.ts.snap

```diff
@@ -92,6 +92,19 @@ To enable this feature on this environment, use:
 `;
 
 exports[`targeting update updates a target in headless mode 1`] = `
-"{\\"_feature\\":\\"647e36gg16d9k4815fi3f97d\\",\\"_environment\\":\\"648b421gg3f682869c0f22f8\\",\\"_createdBy\\":\\"test\\",\\"status\\":\\"inactive\\",\\"updatedAt\\":\\"2023-06-27T03:19:58.541Z\\",\\"targets\\":[{\\"_id\\":\\"647e36gg16d9k2814fc3f15e\\",\\"audience\\":{\\"name\\":\\"All Users\\",\\"filters\\":{\\"operator\\":\\"and\\",\\"filters\\":[{\\"type\\":\\"all\\"}]}},\\"distribution\\":[{\\"_variation\\":\\"647e36gg16c1d4814fe3f962\\",\\"percentage\\":1}]}],\\"readonly\\":false}
+"└─ test-env
```

Comment:
is it expected that commands output as a tree in headless mode? I thought that was not the case

Reply:
you're right, it should not be output as a tree in headless mode. Already reverted this change